amm/contracts/pair/lib.rs

```diff
@@ -290,11 +291,12 @@ pub mod pair {
         }
 
         #[ink(message)]
-        fn get_reserves(&self) -> (u128, u128, u32) {
+        fn get_reserves(&self) -> (u128, u128, u32, u8) {
```

Comment:
As discussed on Slack - can we not change this API and have the router call `get_fee` when adding pair to its cache? We already want the Router to always have the pair in its cache, we can cache the fee as well.

Reply:
It's not pretty, but here you go :)